sdk/core/core-amqp/ThirdPartyNotices.txt

```diff
@@ -0,0 +1,321 @@
+Third Party Notices for core-amqp
```

Comment:
Needs notice for util shim (looks like another Joyent/MIT)

Reply:
Needs notice for rhea (Red Hat Inc./Apache2)

Reply:
This file has util here: https://github.com/Azure/azure-sdk-for-js/pull/5843/files/0e8550e84f967c5fbeb4c86ec09722e7f80228b3#diff-e5a49e9f12e090de1e1c94b7e925064cR300

Am I missing something? (I'm adding the transitive dependencies).